src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -85,7 +109,7 @@ public void toStringMethod() {
     /**
      * Parses {@code userInput} into a {@code NameContainsKeywordsPredicate}.
```

Comment:
Should this be changed to `Parses {@code userInput} into a {@code NameNricContainsKeywordsPredicate}.`

Reply:
Yeah it should. I missed that out thanks for catching it